elasticdl/cc/common/tensor.h

```diff
@@ -0,0 +1,72 @@
+#pragma once
```

Comment:
Should we use `gragma once` or include guards? Seems that the once difference is that if the header file is copied around different locations by the compiler, `gragma once` won't work.

Reply:
I have no preference for using `#pragma once` or include guards. If google style suggests include guards, I will follow it.